### PR TITLE
Feat/cordova

### LINKED ISSUE #551: Modify app for cordova
Client Side:

- [x] Refactor Meditation Bell with native notifications
- [ ] ~Refactor Push Messaging/Service Worker using FCM~ (moving to own issue)
- [x] Remove unused assets, update `index.html`

### PR DESCRIPTION
**Pull request by olastor**
- modified app for cordova
- added native notifications for sessions

closes #551

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,14 +20,21 @@ export let ApiConfig = {
...
 ```
-yarn
-ng serve
+
+### Building
+```bash
+$ yarn build # add "--prod" for production build
 ```
 
+The output directory by default is `dist/`. To build for cordova add the command line argumen `--app cordova`.
```
Comment (binarious) [inline README.md]: argumen*t*

File: src/push-service.js [hunk 2]
```diff
@@ -1,76 +1,27 @@
...
+// Give the service worker access to Firebase Messaging.
+// Note that you can only use Firebase Messaging here, other Firebase libraries
+// are not available in the service worker.
+importScripts('https://www.gstatic.com/firebasejs/4.5.0/firebase-app.js');
+importScripts('https://www.gstatic.com/firebasejs/4.5.0/firebase-messaging.js');
+
+// Initialize the Firebase app in the service worker by passing in the
+// messagingSenderId.
+firebase.initializeApp({
+  'messagingSenderId': '223027225088'
```
Comment (binarious) [inline src/push-service.js]: What's this id?

**Review by binarious — CHANGES_REQUESTED** [review]

File: src/push-service.js (hunk 2 above)
Comment (olastor) [reply src/push-service.js]: This branch is not really ready yet at all :D Will need more time for developing and testing.

File: src/app/profile/flag/flag.component.ts [hunk 3]
```diff
@@ -1,27 +1,39 @@
...
+      "
+      src="assets/img/flags/16/{{ countryCode.toLowerCase() }}.png"
+    >
   `
 })
 export class FlagComponent {
 
   @Input() countryCode: string;
 
+  isCordova = typeof cordova !== 'undefined';
```
Comment (binarious) [inline src/app/profile/flag/flag.component.ts]: Please add:
```ts
export const IS_CORDOVA = typeof cordova !== 'undefined';
```
to `app.module.ts` or something else and import the constant from there in every component where it's needed.

File: src/index_cordova.html [hunk 4]
```diff
@@ -0,0 +1,88 @@
+<!doctype html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Meditation+</title>
+  <base href=".">
+
+  <!--
+    Content-Security-Policy:
```
Comment (binarious) [inline src/index_cordova.html]: Is this comment actually needed?

**Review by binarious — CHANGES_REQUESTED** [review]

File: src/index_cordova.html (hunk 4 above)
Comment (olastor) [reply src/index_cordova.html]: No, was only for having a better overview. I will remove it.

**Comment by binarious** [comment]
Why did you add the wip label again?

**Comment by olastor** [comment]
@binarious I didn't test the changes in the latest Commits yet. Don't know If it builds with cordova.

**Comment by binarious** [comment]
Development moved to https://gitlab.com/sirimangalo/meditation-plus

**Pull request closed without merging**